credentialdigger/models/snippet_model/snippet_model.py

```diff
@@ -76,17 +77,24 @@ def analyze(self, discovery):
             # since either the snippet is empty or there is just one word
             return True
 
-        if len(data) == 2:
-            # No need to pre-process data since we have only two words
-            # Predict if the string `word1 + ' ' + word2` is a false positive
-            label = self.model.predict(
-                data[0] + ' ' + data[1])[0]  # 0=label, 1=probability
+        # Extract the committed secret
+        index_of_value = self._label_preprocess(data)
```

Comment:
Place a comment here to keep track of the format of this variable, otherwise we'll forget it (more on this in the command of L92)